src/index.html

```diff
@@ -10,13 +10,98 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute should describe the image content, not just say 'logo'. For example, 'Company logo' would be more descriptive.